### PR TITLE
Add license.

### PR DESCRIPTION
**Pull request by kaikai1024**
* Use erc20 from openzepplin
* Add Apache 2.0 license

TODO: update cita/remove-gpl

### CODE REVIEW

**Review by leeyr338 — DISMISSED** [review]
It is okay to OpenZeppelin .

**Comment by kaikai1024** [comment]
DONE @leeyr338

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (leeyr338) [inline LICENSE]: Add [name of copyright owner] likes:
```
Copyright Cryptape Technologies LLC.
```
And as @rainchen 's suggestion, we do not need the time now.

**Review by leeyr338 — COMMENTED** [review]
I not sure it's okay to move 3rd_party_license MIT to Apache-2.0

File: LICENSE (hunk 1 above)
Comment (kaikai1024) [reply LICENSE]: ok.
Maybe we should add a file naming `COPYING`

**Review by leeyr338 — APPROVED** [review]

**Pull request merged**